master_changes.md

```diff
@@ -21,6 +21,7 @@ New option/command/subcommand are prefixed with ◈.
   * Don't patch twice [#4529 @rjbou]
   * With `--deps-only`, set dependencies as root packages [#4964 @rjbou - fix #4502]
   * Keep global lock only if root format upgrade is performed [#4612 @rjbou - fix #4597]
+  * Only track `.install` content when install field is empty instead of whole switch directory, makes install faster [#4494 @kit-ty-kate @rjbou - fix #4422]
```

Comment:
I can't parse this sentence :rofl: 
```suggestion
  * Improve installation times by only tracking files listed in `.install`  instead of the whole switch prefix when there are no `install:` instructions [#4494 @kit-ty-kate @rjbou - fix #4422]
```